SETUP.md

```diff
@@ -4,8 +4,8 @@ The back-end is a full Rails project, but the expectation is that you will set u
 
 ## Environment Setup
 
-1. Install the project Ruby version `2.5.1`
-2. Install Node 8 or higher (the project expects `8.16.0`)
+1. Install the project Ruby version `2.6.5`
+2. Install Node 12 or higher (the project expects `12.16.2`)
```

Comment:
We should really change this to just `install Node 16.12.2` as the package.json will complain if you don't install the exact version.